mg400_bringup/launch/display.launch.py

```diff
@@ -13,47 +13,40 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from pathlib import Path
-from typing import Dict
+# from pathlib import Path
+# from typing import Dict
 
 from ament_index_python import get_package_share_path
+import h6x_bringup_common.config_loader as cl
```

Comment:
New implicit dependencies.
https://github.com/HarvestX/MG400_ROS2/blob/humble/build_depends.repos

The extenal useres can not use this launch script since this extenal dependencie is not described in the script or README.

Please think of hard-coding xacro perser as like before.

Reply:
change to hard-coding xacro parser.